CONTRIBUTING.md

```diff
@@ -0,0 +1,138 @@
+# Code review checklist (TODO: should be copy-pasted in PR template)
+
+- [ ] code should be formatted using `rustfmt` (:warning: this also applied to code in macro like `tokio::select!` and `lazy_static!`
+- [ ] code should be exempt of `clippy` lints (and function size should not exempt 50 lines)
+- [ ] code should not `panic!` an `Err` variant should be propagated and capture by top-level `async main() -> Result<()>`
```

Comment:
Maybe add a link to a doc about how we manage errors. (I'll try to come back later and update this comment with a link)

Reply:
```suggestion
- [ ] code should not `panic!` an `Err` variant should be propagated and capture by top-level `async main() -> Result<()>`

In addition, as a reviewer, if you can make a code suggestion to accelerate the process it'll be greatly appreciated!
```

(so meta)

Reply:
Same remark

Reply:
@yvan-sraka I think you can lick the upper button to merge it if you are agree !